integration_tests/suite/helpers/database.py

```diff
@@ -1,19 +1,48 @@
-# Copyright 2017 The Wazo Authors  (see the AUTHORS file)
+# Copyright 2017-2019 The Wazo Authors  (see the AUTHORS file)
 # SPDX-License-Identifier: GPL-3.0-or-later
 
+from functools import wraps
 import logging
+import uuid
 import sqlalchemy as sa
 
 from contextlib import contextmanager
 from sqlalchemy.orm import sessionmaker
-from sqlalchemy.orm.session import make_transient
 from sqlalchemy.sql import text
 from xivo_dao.alchemy.call_log import CallLog
 from xivo_dao.tests.test_dao import ItemInserter
 
+from .constants import VALID_TENANT
+
 logger = logging.getLogger(__name__)
 
 
+def call_logs(call_logs):
+    def _decorate(func):
+        @wraps(func)
+        def wrapped_function(self, *args, **kwargs):
+            with self.database.queries() as queries:
+                for call_log in call_logs:
+                    participants = call_log.pop('participants', [{
+                        'user_uuid': str(uuid.uuid4()),
+                        'tenant_uuid': VALID_TENANT,
+                        'role': 'source',
+                    }])
```

Comment:
Why you need to populate one participants by default (from [previous state](https://github.com/wazo-pbx/wazo-call-logd/blob/master/integration_tests/suite/test_cdr.py#L43))? We try to keep [fixtures](https://github.com/wazo-pbx/wazo-chatd/blob/master/integration_tests/suite/helpers/fixtures/db.py) with the minimal parameters possible to exist. If the test need more, then it should define it explicitly

Reply:
Since you can only retrieve cdr that belong someone in REST API, all tests need at least one participants (except maybe one to show the cdr without participants doesn't show up in API).

Reply:
(just thinking out of loud) 

Or `a call_lod should also have a tenant_uuid, because all calls are from/to a tenant even if we don't known the participants yet`:
* if that false, the current implementation is ok
* if that true we need to find a how to link the `cel` with a `tenant_uuid` without using `participants`.

I haven't found this link with the information I have in a `cel`, do I miss something ?

Reply:
For me all cel should have a tenant. With or without participants. In cel we have the context that you can use to link with a tenant